src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/Junit5FunctionalSpec.groovy

```diff
@@ -11,8 +11,10 @@ class Junit5FunctionalSpec extends AbstractPitestFunctionalSpec {
         given:
             copyResources("testProjects/junit5kotlin", "")
         when:
-            ExecutionResult result = runTasksSuccessfully('pitest')
+            ExecutionResult result = runTasks('pitest')
```

Comment:
Nice! They most likely was not available when I started to use nebula-test years ago. It's must easier to determine a reason of (unexpected) failure.